### ISSUE TITLE
Questions array in YAML file?

### ISSUE THREAD

**Issue by jlyon** [GovReady/security-stories-nist800-53#3]
@gregelin  at the bottom of https://github.com/GovReady/govready-q-ssp-module/blob/master/modules/au_80053_audit_set1.yaml there is an empty  array:
```
questions: []
```

Should we be filling this in with anything, or is this just needed to match the SSP yaml file syntax?

**Comment by gregelin** [comment]
@JoshData There appears to be a trailing `questions: []` parameter in https://github.com/GovReady/govready-q-ssp-module/blob/master/modules/au_80053_audit_set1.yaml that is not found in https://github.com/GovReady/Issue-Packs/blob/master/examples/au_80053_audit_set1.yaml or other examples from IssuePacks. Do you know why it is there? Thanks!

**Comment by JoshData** [comment]
Oh, sorry, I screwed it up. Q is trying to load the IssuePack YAML file as if it were a Q module, because it has a yaml file extension. But it wasn't valid for Q, so it was generating errors in load_modules. I added some lines in https://github.com/GovReady/govready-q-ssp-module/commit/9be541446df39f4bf11383dc8566c8656f850d34 to make it able to be loaded as a module. I didn't realize when I made that commit that I was editing the IssuePack YAML file.

The YAML file needs to move out of the directory tree Q scans for modules, or we need to add a way to exclude it from being loaded as a module.
